test/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Running the tests
 
-This folder contains test cases for HWI. To run these tests, `hwilib` will need to be installed to your python system. You can install it by doing `pip install -e .[tests]` in the root directory.
+This folder contains test cases for HWI. To run these tests, `hwilib` will need to be installed to your python system. You can install it by doing `pip install -e .[tests]` in the root directory. In addition install `pip install hidapi python-bitcoinrpc`.
```

Comment:
Instead of listing specific dependencies, I think we should take the approach that was done in the main readme Just tell people to go look in `pyproject.toml` and have them install everything under `[tool.poetry.dependecies]` and `[tool.poetry.dev-dependencies]`.

Also `[tests]` is no longer an install target, so we can drop that. So the command for installing hwilib will be `pip install -e .`.